specification/ai/data-plane/Azure.AI.Agents/stable/2025-05-01/azure-ai-agents.json

```diff
@@ -1094,6 +1094,43 @@
             "$ref": "./examples/Messages_UpdateMessage_MinimumSet_Gen.json"
           }
         }
+      },
```

Comment:
Your change should only apply to the "v1" and "-preview" versions, not to this version.

Reply:
Fixed